src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -440,6 +440,50 @@ namespace winrt::TerminalApp::implementation
         control.ScrollViewport(::base::ClampAdd(currentOffset, delta));
     }
 
+    // Method Description:
+    // - Serializes the state of this tab as a series of commands that can be
+    //   executed to recreate it.
+    // Arguments:
+    // - <none>
+    // Return Value:
+    // - A vector of commands
+    std::vector<ActionAndArgs> TerminalTab::BuildStartupActions() const
+    {
+        // Give initial ids (0 for the child created with this tab,
+        // 1 for the child after the first split.
+        auto state = _rootPane->BuildStartupActions(0, 1);
+
+        ActionAndArgs newTabAction{};
+        newTabAction.Action(ShortcutAction::NewTab);
+        NewTabArgs newTabArgs{ state.firstPane->GetTerminalArgsForPane() };
+        newTabAction.Args(newTabArgs);
+
+        state.args.insert(state.args.begin(), newTabAction);

```

Comment:
I would disagree with _always_ since you might not always want the move constructor, but that is just being pedantic :)

Reply:
In this case trying to make the change for <ConversionTrait<std::vector<T>>  I couldn't use emplace_back or the push_back with rvalue ref specifically because the autogenerated classes from the .idl files don't have move constructors so I had to revert that one change.